examples/sft/gsm8k/run_qwen_05_peft.sh

```diff
@@ -0,0 +1,35 @@
+# Tested with 2 & 4 GPUs
+
+set -x
+
+if [ "$#" -lt 2 ]; then
+    echo "Usage: run_gemma_2b.sh <nproc_per_node> <save_path> [other_configs...]"
```

Comment:
A typo here, should be run_qwen_05_peft.sh

Reply:
fixed!